### DISCUSSION TITLE
Image optimization without alt tag?

### DISCUSSION THREAD

**Discussion by abroadwin** [vercel/next.js#59132] [Help]
### Summary

While testing a new site I'm working on using Google Lighthouse I ran into the following warning:

> **Image elements have [alt] attributes that are redundant text.**
> Informative elements should aim for short, descriptive alternative text. Alternative text that is exactly the same as the text adjacent to the link or image is potentially confusing for screen reader users, because the text will be read twice

It's true that in this particular case (a logo next to the company name) the image is right next to text that suffices as the alt for that image. Unfortunately I know of no way to take advantage of the features of the Image tag without providing an alt. Is there any way to do so?

All I can think of is to provide a different alt text for the image, but in this case by far the nicest experience for those using screen readers would actually be not having any alt text at all.

Thanks in advance for any time or help.

Best,
Alex

**Comment by joshh20** [comment]
Just ran into the same situation. As next/image requires the alt tag, it makes it kind of difficult. For now, I just used an empty string.
